src/libs/PolicyUtils.ts

```diff
@@ -575,6 +575,40 @@ function getIntegrationLastSuccessfulDate(connection?: Connections[keyof Connect
     return (connection as ConnectionWithLastSyncData)?.lastSync?.successfulDate;
 }
 
+function getSageIntacctVendors(policy: Policy | undefined, selectedVendorId: string | null | undefined): SelectorType[] {
```

Comment:
Can't we just do this?

```suggestion
function getSageIntacctVendors(policy?: Policy, selectedVendorId?: string): SelectorType[] {
```